Model/ResourceModel/Events.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Klaviyo\Reclaim\Model\ResourceModel;
+
+use Klaviyo\Reclaim\Setup\SchemaInterface;
+use Magento\Framework\Model\ResourceModel\Db\AbstractDb;
+
+/**
+ * Klaviyo Events Table ResourceModel.
+ *
+ * The ResourceModel for any Model has the ability to query and make transactions with its associated table in the database.
+ * This queries through the Zend\Db adapter which is connected to the Databse using the etc/env.php file.
+ * The ResourceModel requires the Model to create DataObject instances and requires the table name and its idFieldName
```

Comment:
Can you elaborate on this? I'm not explicitly seeing the connection to the Events Model here. Can you either explain  the relationship between the ResourceModel and the Model or link to Magento documentation on this topic? I'm asking so we can have this code be self-documenting such that someone in the future can look at these event classes, be able to understand exactly how they interact (and interact with the Magento core code) and then implement a new table.